classifier_and_summarizer_interface.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, contents, max_pages):
         # object of type StampPages
         self.stampified_pages = None
 
-    def preprocess_contents(self):
+    def _preprocess_contents(self):
```

Comment:
Why does the file name contain Interface?

Reply:
Will update accordingly - this is not necessary